api/src/main/resources/application.yml

```diff
@@ -30,4 +31,13 @@ security:
       userInfoUri: https://api.twitch.tv/helix/users
       preferTokenInfo: true
 
-frontEndUrl: http://localhost:3000
\ No newline at end of file
+frontEndUrl: http://localhost:3000
+logging:
```

Comment:
add a blank line between fronendUrl and the logging block. The whtie space is a nice way to keep separate ideas/areas-of-config isolated from each other